config/locales/sufia.en.yml

```diff
@@ -1,6 +1,7 @@
 en:
   sufia:
     account_name:       "My Institution Account Id"
+    account_label:      "User"
```

Comment:
Actually, this isn't related to this story.  I just noticed that it was missing when I was looking at the permissions tab when editing a generic file:
https://github.com/projecthydra/sufia/blob/master/app/views/generic_files/_permission_form.html.erb#L44

I should probably take it out of this PR and submit it separately.